### PR TITLE
[test] Port GPUDIRECTACC check to Tiger

### PR DESCRIPTION
**Pull request by jgphpc**
Will fix UES-567

### CODE REVIEW

**Comment by jgphpc** [comment]
> unittests/test_schedulers.py:402: AssertionError

File: cscs-checks/mch/gpu_direct_acc.py [hunk 1]
```diff
@@ -11,9 +12,14 @@ def __init__(self):
...
-        if self.current_system.name in ['daint', 'dom']:
+        if self.current_system.name in ['daint', 'dom', 'tiger']:
             self.modules = ['craype-accel-nvidia60']
-            self.variables = {'MPICH_RDMA_ENABLED_CUDA': '1'}
+            self.variables = {
+                'MPICH_RDMA_ENABLED_CUDA': '1',
+                # this is mandatory to have the Cray wrapper compiling
+                # OpenACC codes correctly:
+                'PATH': '%s:$PATH' % os.environ['CRAY_BINUTILS_BIN']
+            }
```
Comment (vkarak) [inline cscs-checks/mch/gpu_direct_acc.py]: How this wasn't needed on Daint/Dom?

Comment (vkarak) [reply cscs-checks/mch/gpu_direct_acc.py]: @jgphpc Could you elaborate a bit on that?

**Comment by jgphpc** [comment]
@jenkins-cscs retry dom

File: cscs-checks/mch/gpu_direct_acc.py (hunk 1 above)
Comment (jgphpc) [reply cscs-checks/mch/gpu_direct_acc.py]: If i don't, the check will fail on tiger with:
```
ATP Stack walkback for Rank 0 starting:
  _start@start.S:120
  __libc_start_main@0x2aaaafc25f89
  main@0x4014a2
  MAIN_@gpu_direct_acc.F90:26
  pmpi_reduce_@0x2aaaac38b319
  PMPI_Reduce@0x2aaaac2d828c
  MPIR_Reduce_impl@0x2aaaac2d5410
  MPIR_Reduce_intra@0x2aaaac2d63f7
  MPIR_Localcopy@0x2aaaac394a03
  _cray_mpi_memcpy_snb@0x2aaaac49cc58
ATP Stack walkback for Rank 0 done
Process died with signal 11: 'Segmentation fault'
```

> stat-view atpMergedBT.dot

eff

Comment (vkarak) [reply cscs-checks/mch/gpu_direct_acc.py]: If it's Tiger-specific, I'd suggest to add it only for that system.

**Pull request merged**